vignettes/miscellaneous-examples.Rmd

```diff
@@ -32,7 +32,7 @@ library(ggplot2)
 
 ```{r}
 ggpk_dot_matrix <- function(..., width = 10) {
-  ggpacket(...) %+%
+  ggpacket() %+%
```

Comment:
I'll need to explore this change and why propagating ellipses args it's problematic. It's a pretty important part of the API of the package. 

However, please don't take that to be a concern to `ggplot2`. This package was built before any of the `tidyeval` stuff was introduced and I was surprised to see that it survived that transition with minimal changes. The landscape has changed a lot since then and if it's starting to show cracks I think that might mean this package may need to modernize.

Reply:
Well the error message I get when running this example is the following:

> Error in coord_cartesian(ratio = ratio, ...) : unused argument (size = 3)
Error during wrapup: not that many frames on the stack
Error: no more error handlers available (recursive errors?); invoking 'abort' restart

Which would suggest to me that the coordinate system gets fed the wrong argument. I have no idea about how the plumbing works though, so I might be overlooking something.

Reply:
Ah cool, that might be an easier fix than I initially thought. I probably just need to update the example used [here](https://github.com/dgkf/ggpackets/blob/0279f3a734870768f0f648dcd3080f2c4858086a/vignettes/miscellaneous-examples.Rmd#L58) to reflect a change in the accepted function parameters.

I got it from here! I'm sure you have a lot on your plate with the rollout of a major version. Thanks for the PR!